lambda/MAS/Services/StaticContentWriterService.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewEngines;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.Extensions.Logging;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace MAS.Services
+{
+    public interface IStaticContentWriter
```

Comment:
I'm not sure that IStaticContentWriter is a good name, as it's not writing anything. Maybe view IViewRenderer or something?